rwengine/src/engine/GameData.cpp

```diff
@@ -49,10 +38,15 @@ GameData::~GameData()
 
 void GameData::load()
 {
+	index.indexGameDirectory(datpath);
 	index.indexTree(datpath);
-	
-	parseDAT(datpath+"/data/default.dat");
-	parseDAT(datpath+"/data/gta3.dat");
+
+	// Initalize all the archives.
+	loadIMG("models/gta3.img");
+	loadIMG("anim/cuts.img");
```

Comment:
Needs a note that this shouldn't be considered for loading files in the same way as gta3.img